_oasis

```diff
@@ -185,7 +185,7 @@ Library cli
                     CLIPager
   InternalModules:  CLIData, CLIPluginsLoaded
   CompiledObject:   byte
-  BuildDepends:     oasis, oasis.base, plugin-loader
+  BuildDepends:     oasis, oasis.base, plugin-loader, bytes
```

Comment:
Mmmh no, I forgot, it's even included in ocamlfind >= 1.5.2 so there is no extra dependency.